src/components/App.js

```diff
@@ -1,33 +1,26 @@
 import React, { useState, useEffect } from 'react';
-// getAPIHealth is defined in our axios-services directory index.js
-// you can think of that directory as a collection of api adapters
-// where each adapter fetches specific info from our express server's /api route
-import { getAPIHealth } from '../axios-services';
-import '../style/App.css';
+import { Route, Switch, Link } from 'react-router-dom';
 
-const App = () => {
-  const [APIHealth, setAPIHealth] = useState('');
-
-  useEffect(() => {
-    // follow this pattern inside your useEffect calls:
-    // first, create an async function that will wrap your axios service adapter
-    // invoke the adapter, await the response, and set the data
-    const getAPIStatus = async () => {
-      const { healthy } = await getAPIHealth();
-      setAPIHealth(healthy ? 'api is up! :D' : 'api is down :/');
-    };
+/* 
+IMPORT REACT COMPONENTS BELOW
+*/
+import { AccountForm, Navbar } from '../components'
 
-    // second, after you've defined your getter above
-    // invoke it immediately after its declaration, inside the useEffect callback
-    getAPIStatus();
-  }, []);
+const App = () => {
 
   return (
-    <div className="app-container">
-      <h1>Hello, World!</h1>
-      <p>API Status: {APIHealth}</p>
-    </div>
+    <>
+      <Navbar />
+      <Switch>
```

Comment:
When we have class or whenever you're free doing the day, could you let me know what is "switch" is doing for app? Otherwise, looks good.